metadata-transformer/src/main/resources/context.json

```diff
@@ -5,6 +5,9 @@
     "type": "@type",
     "graph": "@graph",
     "value": "@value",
+    "doi": {
```

Comment:
DOIs are actually URIs, this keeps the datatype intact in the framed JSON-LD, but presents it as a simple value